pkg/apis/akash.network/v1/register.go

```diff
@@ -0,0 +1,68 @@
+package v1
+
+import (
+	"reflect"
+
+	apiextv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apiextcs "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	CRDPlural   string = "manifests"
+	CRDGroup    string = "akash.network"
+	CRDVersion  string = "v1"
+	FullCRDName string = CRDPlural + "." + CRDGroup
+)
+
+var (
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	AddToScheme   = SchemeBuilder.AddToScheme
+)
+
+// Create a  Rest client with the new CRD Schema
```

Comment:
comment on exported var SchemeGroupVersion should be of the form "SchemeGroupVersion ..."